examples/carousel.js

```diff
@@ -24,5 +24,12 @@ export default class CarouselSample {
           allowLooping: true,
         });
     CarouselSynchronizer.getSingleton(this).sync(a, b);
+    const c = new Carousel(
+        document.querySelector('.carousel--c'),
+        Array.from(document.querySelectorAll('.carousel--c .slide')),
+        {
+          allowLooping: false,
+          transition: new DraggableSlide(),
```

Comment:
No action needed.

For future, It'd be awesome if you can expand the examples with other transitions and other settings.